apps/taxes/src/modules/avatax/avatax-line-number-resolver.ts

```diff
@@ -0,0 +1,8 @@
+import { OrderLineFragment } from "../../../generated/graphql";
+
+/**
+ * @description Makes sure we use the same line number for creating & refunding transaction. AvaTax requires that the line number is <= 50 characters.
+ */
+export function resolveAvataxTransactionLineNumber(line: OrderLineFragment) {
+  return line.id.substring(0, 50);
```

Comment:
Saleor IDs look like this:

```
OrderLine:<uuid>
```

I think we should take 50 last characters from base64, since they are the uuid of the item, not the prefix.

For example:
![CleanShot 2024-01-31 at 11 31 07@2x](https://github.com/saleor/apps/assets/4144459/791bd222-60b3-4d9d-8e70-45a9fcab5f9e)